rollbar/src/test/java/com/rollbar/RollbarTest.java

```diff
@@ -0,0 +1,21 @@
+package com.rollbar;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+
+public class RollbarTest {
+
+    @Test
+    public void itDoesNotThrowAnExceptionWhenConfiguringWithJustAccessTokenAndEnvironment() {
+        Boolean exceptionDidHappen = false;
+
+        try {
+            new Rollbar("some-access-token", "some-environment");
+        } catch (NullPointerException e) {
+            exceptionDidHappen = true;
```

Comment:
You can just use `fail()` here, and then drop the `exceptionDidHappen` field.

Also you might be better off with:

```java
Rollbar r = new Rollbar("Some access token", "environment name");
pass();
```

The uncaught error will propagate up and will display better in the test panels/command line. (Because it will include the exception stack trace).